front/listeners/procuration-reason.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * Display special message if the reason for procuration is 'residency'
+ */
+export default () => {
+    const dropdown = dom('#app_procuration_elections_reason');
```

Comment:
You shouldn't put this in a listener as it would be loaded on every page. Instead, you could put this code in the procurations pages files `front/pages/procuration*.js`